docs/madr/1-intial.md

```diff
@@ -0,0 +1,155 @@
+# An event driven ANT registration system in AO to optimize ANT listings by access role
+
+- Status: accepted
+- Deciders: [Dylan], [Ariel], [Phil]
+- Date: 2024-07-22
+- Authors: [Atticus], [Dylan], [Ariel], [Phil]
+
+## Context and Problem Statement
+
+Listing all ANTs registered to ArNS names and owned by a specific owner or
+owners becomes more computationally expensive as more ArNS names are registered.
+Additionally, its computationally infeasible to find ANTs not registered to an
+ArNS. GraphQL-based approaches cannot currently support either of these
+scenarios.
+
+We need to provide owner-based lookups that perform better than O(N).
+
+This is done both on the client side via [AoConnect], or as inter-process
+communication (like in the [ArNS Resolver](#arns-resolver)).
+
+### Additional Background
+
+The ArNS AO Process is responsible for recording which ANTs are registered to
+each ArNS name. It does not store or track additional information about the
+owner of the ANT. This reduces the amount of state managed, providing more
+concurrency in the operations in the overall name system's design, (e.g.
+individual ANT's activities do not impact performance of the ArNS Process).
+
```

Comment:
i don't see any mention of _why_ we have ANTs in the first place. may be worth mentioning why ownership information is stored in an ANT vs. the registry itself.